src/lib/support_config.ts

```diff
@@ -181,7 +181,7 @@ Device: ${submission.device}`;
 Submitted by: ${user.name}`;
 
         } else {
-            desc = `${desc}\n\nReason and urgency:\n${submission.reason}`;
+            desc = `${desc}\n\nReason:\n${submission.reason}\n\nUrgent:\n${submission.urgency}`;
```

Comment:
```suggestion
            desc = `${desc}\n\nReason:\n${submission.reason}\n\nUrgent: ${submission.urgency}`;
```
I think even only one new line between "reason" and "urgent" (`${submission.reason}\nUrgent:`) may be enough but let's see..